http.bridge/itest/pom.xml

```diff
@@ -83,23 +82,23 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-dependency-plugin</artifactId>
-                <version>2.9</version>
-                <configuration>
-                    <artifactItems>
-                        <artifactItem>
-                            <groupId>org.kaazing</groupId>
-                            <artifactId>netx.http.bridge</artifactId>
-                            <version>${project.version}</version>
-                            <type>jar</type>
-                            <destFileName>netx.http.bridge.jar</destFileName>
-                        </artifactItem>
-                    </artifactItems>
-                </configuration>
                 <executions>
                     <execution>
+                        <phase>package</phase>
```

Comment:
Maven weirdness.  Since I define a phase for the maven-dependency-plugin for the license, I have a feeling that that phase was (for some unknown reason) being used for this execution as well.  I did a google search for the default phase for copy and added it.